src/constants/externalResources.ts

```diff
@@ -102,3 +104,35 @@ export const getParsleyBuildLogURL = (buildId: string) =>
 
 export const getDistroPageUrl = (distroId: string) =>
   `${getUiUrl()}/distros##${distroId}`;
+
+export const honeycombBaseURL = `https://ui.honeycomb.io/mongodb-4b/environments/${
+  isProduction() ? "production" : "staging"
```

Comment:
Considering we have 3 environments prod, staging, and dev that are supported on Honeycomb could we make this a little more generic by using env variables? You can add them to the `.env-cmdrc.json` file. As well as a few other places in project settings,and  [here](https://github.com/evergreen-ci/spruce/blob/6c9f4f3b5bb9270f18d2eaf022e4a439fe307057/.evergreen.yml#L361) and [here](https://github.com/evergreen-ci/spruce/blob/6c9f4f3b5bb9270f18d2eaf022e4a439fe307057/scripts/setup-credentials.js). It would also avoid hardcoding our honeycomb org in the codebase

Reply:
Done.
When this is ready to merge I'll add it to the shared `.env-cmdrc.json`. Will I need to let everyone know to pull it?